chia/rpc/wallet_rpc_api.py

```diff
@@ -1397,13 +1397,14 @@ async def pw_absorb_rewards(self, request) -> Dict:
         if await self.service.wallet_state_manager.synced() is False:
             raise ValueError("Wallet needs to be fully synced before collecting rewards")
         fee = uint64(request.get("fee", 0))
+        max_spends_in_tx = request.get("max_spends_in_tx", None)
```

Comment:
I would like opinions on the name `max_spends_in_tx`.

Should we mention we are in a claim? It's obvious from the context of the API call name `pw_absorb_rewards`.
We may also want to use this parameter or something similar in other calls.